processor/routingprocessor/extract_test.go

```diff
@@ -66,6 +67,34 @@ func TestExtractorForTraces_FromContext(t *testing.T) {
 			fromAttr:      "X-Tenant",
 			expectedValue: "globex",
 		},
+		{
+			name: "value from existing HTTP attribute",
+			ctxFunc: func() context.Context {
+				return client.NewContext(context.Background(),
+					client.Info{Metadata: client.NewMetadata(map[string][]string{
+						"X-Tenant": {"acme"},
```

Comment:
There was a case in the past where a header sent by the client as "X-CloudID" (or something to that effect) was placed in the map as "X-Cloudid". Could you add a case to assert that "X-Tenant" is found when searching for "x-tenant" (ie, the lookup is case insensitive)?

Reply:
Hey @jpkrohling, I added the test case.